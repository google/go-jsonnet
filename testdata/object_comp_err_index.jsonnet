{ [error "xxx"]: 42 for x in [1] }
