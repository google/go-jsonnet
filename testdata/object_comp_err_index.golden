RUNTIME ERROR: xxx
-------------------------------------------------
	testdata/object_comp_err_index:1:4-15	$

{ [error "xxx"]: 42 for x in [1] }

-------------------------------------------------
	<builtin>	builtin function <$objectFlatMerge>

-------------------------------------------------
	During evaluation	


